scripts/github-actions/tests-e2e-fixtures-bigquery

```diff
@@ -20,6 +20,13 @@ set -o pipefail
 REPO_ROOT="$(git rev-parse --show-toplevel)"
 cd ${REPO_ROOT}/
 
-export ONLY_TEST_APIGROUPS="bigquery.cnrm.cloud.google.com"
+APIGROUPS=(
+    "bigquery.cnrm.cloud.google.com"
+    "bigqueryanalyticshub.cnrm.cloud.google.com"
```

Comment:
I'm okay with we splitting this way, though intuitively, I have already disassociated bigquery and other APIs with names starting with `bigquery`.